components/brave_shields/resources/panel/state/hooks.ts

```diff
@@ -43,3 +43,30 @@ export function useSiteSettingsData () {
 
   return { siteSettings, getSiteSettings }
 }
+
+export function useIsExpanded () {
+  const [isExpanded, setIsExpanded] = React.useState<boolean | null>(null)
```

Comment:
This results in a little flash whilst the visibility handler fires and gets and returns the value. Same like we saw with the scrollbar. Perhaps the easiest solve for the majority of cases for cold starts on new windows and when the bubble cache expires, is to simply set the current value at WebUI creation on loadTimeData and use that value for the initial state value on this line?

Reply:
added